### PR TITLE
Release v1.3 - CVPR2024 demo release

### LINKED ISSUE #11: No support for IMX219?
Is there any support for Sony IMX219 image sensors?

### PR DESCRIPTION
**Pull request by taimur-10xe**
Infinite-ISP FPGA Binaries for Kria KV260 Starter Kit with image sensors AR1335, OV5647 and IMX219.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,55 @@ Infinite-ISP Image Signal Processing Pipeline FPGA binaries for XCK26 Zynq® Ult
...
+8. A configuration menu as shown below will appear. Details of the user menu are provided in the [user guide](https://github.com/10x-Engineers/Infinite-ISP_FPGABinaries/blob/main/doc/Infinite-ISP%20Configuration%20Menu.md):
+
+<kbd>![status before dumping](/doc/v1.3/ov5647_user_menu.png)</kbd>
+
+9. If you want to dump frame(s), do so by going to option 2, and then you can remove the SD card from the Kria board.
+
+10. Extract the dumped frame(s) from the SD card and visualize them using the provided [scripts](/scripts).
+
+### For IMX219 image sensor module:
+1. Connect the OV5647 image sensor module to the RPi port on the Kria KV260 AI Starter Kit.
```
Comment (sohaibimran-10xe) [inline README.md:54]: @taimur-10xe IMX219

File: README.md [hunk 2]
```diff
@@ -15,6 +15,55 @@ Infinite-ISP Image Signal Processing Pipeline FPGA binaries for XCK26 Zynq® Ult
...
+7. Connect the USB cable with the JTAG/USB port on Kria to read the messages over the serial interface (baud rate 115200).
+8. A configuration menu as shown below will appear. Details of the user menu are provided in the [user guide](https://github.com/10x-Engineers/Infinite-ISP_FPGABinaries/blob/main/doc/Infinite-ISP%20Configuration%20Menu.md):
+
+<kbd>![status before dumping](/doc/v1.3/ov5647_user_menu.png)</kbd>
+
+9. If you want to dump frame(s), do so by going to option 2, and then you can remove the SD card from the Kria board.
+
+10. Extract the dumped frame(s) from the SD card and visualize them using the provided [scripts](/scripts).
+
+### For IMX219 image sensor module:
```
Comment (sohaibimran-10xe) [inline README.md]: I think we can bring this section to the top (IMX219)

**Review by sohaibimran-10xe — CHANGES_REQUESTED** [review]

**Review by taimur-10xe — COMMENTED** [review]
changes made to ReadMe.md as suggested.

File: README.md (hunk 2 above)
Comment (taimur-10xe) [reply README.md]: changes made as requested.

File: README.md (hunk 1 above)
Comment (taimur-10xe) [reply README.md:54]: correction made.

**Review by sohaibimran-10xe — APPROVED** [review]

**Pull request merged**